packages/next/lib/verifyRootLayout.ts

```diff
@@ -26,7 +26,6 @@ function getRootLayout(isTs: boolean) {
 }) {
   return (
     <html>
-      <head></head>
       <body>{children}</body>
```

Comment:
Should we leave an empty head tag here similar to our default template?
x-ref: https://github.com/vercel/next.js/blob/2e3e56d911cf7772deea1c2266718b9364d6451d/packages/create-next-app/templates/app/js/app/layout.jsx#L6-L10

Reply:
sgtm